docs/docs/changelog.md

```
@@ -4,11 +4,63 @@
+  Due to some events having end datetime > start datetime, the API was throwing an error.
+- **[#1158](https://github.com/fossunited/fossunited/issues/1158)** Search support in Schedule page
+  Search whole Event schedule in flat List based on Talk Title, Speaker name, Designation and category. Also card shows Hall and Date as Indicator
+- **[#1151](https://github.com/fossunited/fossunited/issues/1151)** Schedule Download formats
+  Schedule can be downloaded in various formats now. CSV | ICS | Orgmode | Markdown and so on.
+  Further plan is to make PDF generation in FOSS theme using `typst`
+
+---
+### 🌟 New Contributor Spotlight
+- 🎉 [@agriyakhetarpal](https://github.com/agriyakhetarpal) made their first contribution:
+  - [#1142](https://github.com/fossunited/fossunited/pull/1142): Add Zulip logo for chapters
+	With merge of Zulip into svgl, we now have zulip support into socials link for Chapter Info.
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)